packages/runtime/src/runtimes/react/controls/rich-text.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+  ReactNode,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+  KeyboardEvent as ReactKeyboardEvent,
+  FocusEvent as ReactFocusEvent,
+} from 'react'
+
+import {
+  RichTextControl,
+  RichTextControlData,
+  RichTextControlDefinition,
+} from '../../../controls/rich-text'
+import { RichTextEditor } from '../../../components/builtin/Text/components/RichTextEditor'
+// @ts-expect-error: no types for 'slate-hotkeys'
+import Hotkeys from 'slate-hotkeys'
+import { isHotkey } from 'is-hotkey'
+
+import { Value } from 'slate'
+import { Editor, OnChangeParam } from 'slate-react'
```

Comment:
These imports are quite heavy and I'm worried this could be affecting live page bundles even when a site isn't using this control. Can you please check bundle sizes and figure out how to code-split if it's a problem?

Reply:
I am taking this comment to mean that I should look into reusing the Text component.

Reply:
That sounds like it could solve any bundle size problems, yeah. The Text component might still be included in a site even if the control isn't used, but if it's light enough, that might not be a big deal. So regardless, you probably want to check the bundle and verify